lib/deprecation_toolkit/collector.rb

```diff
@@ -33,7 +33,7 @@ def <=>(other)
     end
 
     def deprecations_without_stacktrace
-      deprecations.map { |deprecation| deprecation.sub(/ \(called from .*\)$/, "") }
+      deprecations.map { |deprecation| deprecation.sub(/\W\s\(called from .*\)$|\s\(called from .*\)$/, "") }
```

Comment:
Needed to change this regex b/c activesupport 4.2.x adds a period at the end of all the deprecations but that was removed in 5.x

Reply:
Can you instead make it a conditional. It will be simpler to remove this code once we decide at some point to drop support of AS 4.2

```ruby
deprecations.map do |deprecation|
  if ActiveSupport.gem_version >= '5.2'
    ...
  else
    ...
  end
end
```